client/pkg/logutil/log_format.go

```diff
@@ -17,19 +17,19 @@ package logutil
 import "fmt"
 
 const (
-	JsonLogFormat    = "json"
```

Comment:
The only concern is this is a public/exported const. Probably we should 
- deprecate the existing name and plan to remove it in next major or minor release and skip the linter for this const;
- add a new const `JSONLogFormat`.

Reply:
Done, I deprecated `JsonLogFormat`.